### PR TITLE
feat: add dashboard tests

### PR DESCRIPTION
**Pull request by cdurai-netapp**
1. Install grafana and prometheus
2. integrate harvest, grafana and prometheus
3. import dashboards
4. validate dashboard details

### CODE REVIEW

File: integration/test/utils/utils.go [hunk 1]
```diff
@@ -147,3 +162,109 @@ func copyFileContents(src, dst string) (err error) {
...
+
+func IsUrlReachable(url string) bool {
+	response, errors := http.Get(url)
+	if errors == nil && response.StatusCode == 200 {
+		return true
+	}
+	return false
+}
+
+func AddPPrometheusToGrafana() {
```
Comment (cgrinds) [inline integration/test/utils/utils.go:174]: Should this be `AddPrometheusToGrafana`

File: integration/test/dashboard_test.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		for index, _ := range params {
+			execParam = append(execParam, params[index])
+		}
+		importOutput := utils.Run("docker", execParam...)
+		log.Println(importOutput)
+	}
+
+}
+
+func (suite *DashboardImportTestSuite) TestImport() {
```
Comment (cgrinds) [inline integration/test/dashboard_test.go:63]: Would be nice to relate this test to the issue in the repo that caused us to add this test

**Review by cgrinds — COMMENTED** [review]
thanks @cdurai-netapp looks good - left one minor comment and one doc ask

**Review by cgrinds — DISMISSED** [review]

**Review by cgrinds — DISMISSED** [review]

File: integration/Jenkinsfile [hunk 3]
```diff
@@ -35,9 +35,12 @@ pipeline {
...
                             steps {
                                 cleanWs()
-                                sh """
+                                sh '''
                                 echo "Cleaned Up Workspace For Project"
-                                """
+                                docker stop $(docker ps -a -q)
+                                docker rm $(docker ps -a -q)
+                                docker rmi $(docker images -q)
+                                '''
```
Comment (rahulguptajss) [inline integration/Jenkinsfile:43]: may be define as a groovy var and then u can reuse it later.

File: integration/Jenkinsfile [hunk 4]
```diff
@@ -118,14 +123,21 @@ pipeline {
...
                                 echo "Installing"
                                 export PATH=$PATH:/usr/local/go/bin
-                                export BUILD_PATH=$DOCKER
+                                #export BUILD_PATH=$DOCKER
+                                cd $WORKSPACE/harvest
+                                cp -rf $WORKSPACE/harvest/integration/test/harvest.yml .
+                                make build
+                                ./bin/harvest stop
+                                ./bin/harvest generate docker full --port --output harvest-compose.yml
+                                docker login -u $DOCKER_USER -p $DOCKER_PWD
```
Comment (rahulguptajss) [inline integration/Jenkinsfile:135]: why do we need to login to docker?

**Review by cgrinds — DISMISSED** [review]

File: integration/Jenkinsfile (hunk 4 above)
Comment (cdurai-netapp) [reply integration/Jenkinsfile]: removed.

File: integration/test/dashboard_test.go (hunk 2 above)
Comment (cdurai-netapp) [reply integration/test/dashboard_test.go:50]: added

File: integration/Jenkinsfile (hunk 3 above)
Comment (cdurai-netapp) [reply integration/Jenkinsfile]: fixed

**Review by cgrinds — APPROVED** [review]

**Pull request merged**